extensions/ql-vscode/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 - Add a `View DIL` command on query history items. This opens a text editor containing the Datalog Intermediary Language representation of the compiled query.
 - Remove feature flag for the AST Viewer. For more information on how to use the AST Viewer, [see the documentation](https://help.semmle.com/codeql/codeql-for-vscode/procedures/exploring-the-structure-of-your-source-code.html).
 - The `codeQL.runningTests.numberOfThreads` setting is now used correctly when running tests.
+- Alter structure of the _Test Explorer_ tree. It now follows the structure of the filesystem instead of the qlpacks.
```

Comment:
```suggestion
- Alter structure of the _Test Explorer_ tree. It now follows the structure of the filesystem instead of the QL packs.
```

Reply:
Whoops I think this suggestion got omitted.